Dockerfile

```diff
@@ -9,7 +9,8 @@ MAINTAINER Rithu John <rithu.john@coreos.com>
 # experience when this doesn't work out of the box.
 #
 # OpenSSL is required so wget can query HTTPS endpoints for health checking.
-RUN apk add --update ca-certificates openssl
+RUN apk add --update ca-certificates openssl xmlsec \
```

Comment:
The library uses external calls. See
https://github.com/Calpicow/go-saml/blob/master/xmlsec.go#L63
https://github.com/Calpicow/go-saml/blob/master/xmlsec.go#L113